### ISSUE TITLE
Installing pcl on windows 10, BOOST error

### ISSUE THREAD

**Issue by anshulvj** [PointCloudLibrary/pcl#2271]
## Your Environment

* Operating System and version: Windows 10 64bit
* Compiler: Cmake 3.11.0: Visual Studio 15 2017 Win64
* PCL Version: 1.8.1 it says on Github

## Context

Trying to build PCL based on instructions given on http://pointclouds.org/documentation/tutorials/compiling_pcl_windows.php#compiling-pcl-windows

Stuck at the "Configure PCL using Cmake" step.

I have installed all dependencies as stated on http://www.pointclouds.org/downloads/windows.html
I presume that I cannot use PCL installer since I am using VS2017, although I can still use the dependencies installers as is given on that link.

## Error

Following is the error when I run Cmake Configure step:

`CMake Error at C:/Program Files/CMake/share/cmake-3.11/Modules/FindBoost.cmake:1956 (message):
  Unable to find the requested Boost libraries.`

`Boost version: 1.50.0`

`  Boost include path: C:/Program Files/Boost/include`

`  Could not find the following static Boost libraries:`

          boost_filesystem
          boost_thread
          boost_date_time
          boost_iostreams
          boost_chrono
          boost_system

`  Some (but not all) of the required Boost libraries were found.  You may
  need to install these additional Boost libraries.  Alternatively, set
  BOOST_LIBRARYDIR to the directory containing Boost libraries or BOOST_ROOT
  to the location of Boost.
Call Stack (most recent call first):
  cmake/pcl_find_boost.cmake:45 (find_package)
  CMakeLists.txt:419 (include)`

I also have Boost environment variables pointing to the boost install directory (installed from the installers stated in above link)

![image](https://user-images.githubusercontent.com/1171507/38262920-ce7ac16c-372b-11e8-9789-a54b58c0cc8a.png)

I read online on other topics that for this error I am supposed to define FindBoost.cmake. But I am presuming PCL CmakeLists files take care of this once the boost environment variables are defined and did not want to mess with adding/changing existing cmake module files with half knowledge. Interestingly inside the `pcl-master/cmake/Modules` there is no FindBoost.cmake but there are other .cmake files such FindEigen.cmake etc.

Any pointers will be helpful. I am almost ready to switch to Linux from Windows if this doesn't work out.

**Comment by UnaNancyOwen** [comment]
The depended binaries and information on that link are so old.
If you want to use pre-built binaries for Visual Studio 2017, You can download [release page](https://github.com/PointCloudLibrary/pcl/releases/tag/pcl-1.8.1).
If you want to know how to build PCL and dependencies from source code, Please see [this documents](https://gist.github.com/UnaNancyOwen/59319050d53c137ca8f3#file-pcl1-8-1-md).

**Comment by adammpolak** [comment]
@anshulvj 

You can point directly at the libraries one at a time, it is a big painful but should take less than 10 minutes. This is the approach I took when running into identical problems and it worked :)

**Comment by hiroMTB** [comment]
I also had a similar problem after I installed All in one package.
It was because cmake's default compiler is Visual Studio 2017 32bit in my environment.
After I made brand new /build directory and configure with Visual Studio 2017 **Win64** like a screenshot, all cmake errors has been disapeared. You dont need to set boost parameter manually.
It's not a bug but not a fantastiac user interface (of cmake)

![config](https://user-images.githubusercontent.com/193280/50478385-19f7dd00-09d1-11e9-95b4-c47c233dfaa6.PNG)
